api/src/main/java/net/kyori/adventure/audience/Audience.java

```diff
@@ -78,6 +80,41 @@
  * @since 4.0.0
  */
 public interface Audience {
+  /**
+   * An empty audience.
+   *
+   * @see #empty()
+   * @since 4.8.0
+   */
+  Key TYPE_EMPTY = Key.key(Adventure.NAMESPACE, "empty");
+  /**
+   * A forwarded audience.
+   *
+   * @see ForwardingAudience
+   * @since 4.8.0
+   */
+  Key TYPE_FORWARDED = Key.key(Adventure.NAMESPACE, "forwarded");
+  /**
+   * A single forwarded audience.
+   *
+   * @see ForwardingAudience.Single
+   * @since 4.8.0
+   */
+  Key TYPE_FORWARDED_SINGLE = Key.key(Adventure.NAMESPACE, "forwarded_single");
+
+  /**
+   * A console audience.
+   *
+   * @since 4.8.0
+   */
+  Key TYPE_CONSOLE = Key.key(Adventure.NAMESPACE, "console");
+  /**
+   * A player audience.
+   *
+   * @since 4.8.0
+   */
+  Key TYPE_PLAYER = Key.key("player");
```

Comment:
note: this is in the `minecraft` namespace, while the others are in the `adventure` namespace - this is because we are using the entity key here, since we can